ui/browser/actions/main-actions.js

```diff
@@ -46,3 +46,7 @@ export function setPageOrder(pageOrder) {
 export function setPageAreaVisibility(visible) {
   return { type: types.SET_PAGE_AREA_VISIBILITY, visible };
 }
+
+export function setWebviewScroll(y) {
```

Comment:
Should we anticipate horizontal and vertical scroll?  We're building an experimentation platform here; we should anticipate doing things we haven't done before, like showing/hiding/blurring sidebars.